tests/webapp_t.py

```diff
@@ -618,6 +619,13 @@ def _assert_tmp_has_no_files(self):
         any_files = any([path.isfile(path.join(tmp, n)) for n in listdir(tmp)])
         self.assertTrue(not any_files, "There are too many files in %s: %s" % (tmp, any_files))
 
+    @patch('loris.transforms.KakaduJP2Transformer.transform')
+    def test_empty_image_file_error(self, mock_transform):
+        to_get = '/%s/full/300,/0/default.jpg' % (self.test_jp2_color_id,)
+        resp = self.client.get(to_get)
+        self.assertEqual(resp.status_code, 500)
+        self.assertTrue('empty derivative file' in resp.data.decode('utf8'))
```

Comment:
I'm not sure we should be exposing this error externally; just logging it somewhere.

Reply:
@alexwlchan should that works for me. I could make just a general server side error.

Reply:
done - pushed a new commit.